### PR TITLE
Add the Our Team page to app-content

### LINKED ISSUE #753: Add remaining team members to Contentful

### PR DESCRIPTION
**Pull request by rogerhutchings**
Adds the Our Team page to app-content

- A lot of this is copied over from the publications page
- Tests are basic, more to come but I wanted to get this out in time for some updates happening (and it's a non-critical page)
- I'm getting some styled-components errors about className not being passed down BUT IT IS

Otherwise, should be good to go.

fix #753, #754, #755 

cc zooniverse/Panoptes-Front-End#5417

# Review Checklist

## General

- [ ] Are the tests passing locally and on Travis?
- [ ] Is the documentation up to date?
- [ ] Is the changelog updated?

## Apps

- [ ] Does it work in all major browsers: Firefox, Chrome, Edge, Safari?
- [ ] Does it work on mobile?
- [ ] Can you `rm -rf node_modules/ && npm install` and app works as expected?

### CODE REVIEW

File: packages/app-content-pages/src/screens/Team/Team.js [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+          people={team.people}
+        />
+      ))}
+    </article>
+  )
+
+  const sidebar = (
+    <Box gap='small'>
+      {filters.map(filter => (
+        <div key={filter.name} >
```
Comment (srallen) [inline packages/app-content-pages/src/screens/Team/Team.js]: Not sure what's best for accessibility, but this sidebar is a menu, so I was thinking initially that this should be an unordered list, but I've read that sometimes lists can be bad for accessibility. @eatyourgreens do you have a suggestion?

**Comment by rogerhutchings** [comment]
> I just noticed that the team for everyone but Chicago is the place, not the institution name. Maybe it should be changed from The Adler Planetarium to Chicago? This is now in Contentful?

It is in Contentful, and this was a straight copy from the existing page - I don't have strong feelings one way or another

File: packages/app-content-pages/src/screens/Team/Team.js (hunk 1 above)
Comment (eatyourgreens) [reply packages/app-content-pages/src/screens/Team/Team.js]: Lists of links should be marked up as lists.

File: packages/app-content-pages/src/screens/Team/components/Person/components/TwitterLink/locales/en.json [hunk 2]
```diff
@@ -0,0 +1,5 @@
+{
+  "TwitterLink": {
+    "a11yTitle": " %(name)s's Twitter feed"
```
Comment (eatyourgreens) [inline packages/app-content-pages/src/screens/Team/components/Person/components/TwitterLink/locales/en.json]: I don’t think this is needed. The link text should be the Twitter username.

**Comment by rogerhutchings** [comment]
At the moment, they're just icon links; we can always tweak this later, I just want to get the bulk of it in.

**Comment by eatyourgreens** [comment]
I missed this in review, but tabbing through the page skips past the filter buttons. The first focussed item is Cliff's twitter link, but I'd expect it to be one of the filter buttons. Tab order should respect the visual layout on the page.

**Comment by eatyourgreens** [comment]
The list of filters should maybe be a radio button group too, since only one can be selected at a time, but that's an enhancement.

**Pull request merged**